src/Subject.ts

```diff
@@ -123,6 +123,12 @@ export class Subject<T> extends Observable<T> implements Observer<T>, Subscripti
     this._complete();
   }
 
+  asObservable(): Observable<T> {
+    return new Observable((observer: Observer<any>) => {
```

Comment:
worst case just cast around it.  If the TypeScript gets in the way of getting things done, work around it. :smile: